### ISSUE TITLE
Application Not Running

### ISSUE THREAD

**Issue by edwinlunando** [id-python/members#41]
I tried to access http://members.python.or.id/ and looks like the server is down?

**Comment by iromli** [comment]
@edwinlunando Yes, the server has been down since months ago (I don't remember exactly). I was trying to seek for new sponsor, but can't find it till now. Another alternative is to host the app in free Paas like Heroku.

**Comment by edwinlunando** [comment]
aku dapet sponsor dari digital ocean nih buat python id. yang bertugas ngurusin deployment siapa? aku kasih credential servernya nanti.

**Comment by iromli** [comment]
@edwinlunando bisa email ke saya @ python.or.id, thanks

**Comment by kusut** [comment]
@edwinlunando is killing it! Thanks for all your work, man!

**Comment by kusut** [comment]
Eh belom live ya? Kirain udah :D

**Comment by edwinlunando** [comment]
Halo semuanya,

Aku mau up-in project members.python.or.id nih. Aku mau bantu deploy. Tapi, aku gak mau deploy ke heroku soalnya aku banyak pengalaman buruk pakai heroku. Solusinya.
1. Pakai server personal punyaku. 
2. Sewa server digital ocean yang 5/10$ per bulan.

Aku prefer yang 1 sih biar gak ribet. Gimana?

**Comment by sugiana** [comment]
Benar, pakai mesin sendiri saja. Apa ada biaya bulanan ?

**Comment by edwinlunando** [comment]
Kalau pakai server saya, gak usah bayar. Kan milik pribadi saya.

—

**Comment by kriwil** [comment]
:+1: re-up

**Comment by edwinlunando** [comment]
Yang punya akses DNS python.or.id siapa ya? Aku sudah deploy nih. Mau rilis pas kopdar. :)

**Comment by iromli** [comment]
@edwinlunando mas Somat

**Comment by edwinlunando** [comment]
@iromli boleh minta kontak mas Somat? Thanks!

**Comment by iromli** [comment]
@edwinlunando cek thread internal milis ya soal kontak
